SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
+
+During the development of the 80001-1 standard, though, it was recognized that risk management is just one of a number of other core themes that had to be managed in concert (e.g., quality management, human factors / usability).  Also ensuring SES required processes that involve a total product lifecycle, with responsibilities transitioning across that period.  To address these requirements, JWG7 developed the <<iso_81001_1_2021>> standard, which also included The Temple diagram to communicate the various aspects of SES that must be considered and managed:
+
+.ISO/IEC 81001-1 _The Temple Diagram_
+
+image::../images/vol1-diagram-81001-temple.svg[algin=center]
+
+#TODO:  SHOULD THE FOLLOWING SOURCE LINE BE A FOOTNOTE ... HUNG ON WHERE ... IHE CONVENTION?#
+
+[none]
+. Source:  <<iso_81001_temple>>
+
+One of the key challenges for implementing this standard, though, is what might be labeld:  *_The Interoperability Trust Gap_*.  This is the technology "hand off" space between the left side of the lifecycle -- Design and development phase, where key responsibility is by each of the "Accountable Manufacturer" organizations, and the right side of the lifecycle -- Implementation phase & Clinical use phase, where key responsibility is on the Accountable Healthcare Delivery Organization (HDO).  Though this reads well in the standards and the model organizes everything in a clear fashion, operationalizing this in real world use remains a Sisyphean effort, primarily due to the amount of expertice, time and resources needed to effectively implement the SES standards as part of normal operating business in HDOs.
+
+To address this SES implementation problem, the SDPi profiles:
+
+. Leverage the ISO/IEEE 11073-1070x Participant Key Purposes (PKP) standards, which represent a consensus standard for risk management of technologies that are implemented in that left-right gap on the Temple model
+. Implementation Conformance Statement (ICS) tables from each of these PKP standards is included in TF-1 Appendix B of this specification, with indication as to whether, how and where each requirement is addressed
+. "Safety, Effectiveness, Security Considerations and Requirements" sections are integrated throughout the profile specifications to link from the PKP ICS table requirements to the satisfying capabilities.
+[none]
+.. Additional non-PKP risk management will also be performed by subject matter experts and formalized in these SES consideration sections, where appropriate.
+
+These SES Considerations sections are the result of the IHE "Security Considerations" sections + the IHE Devices "Safety Considerations" sections, but now consolidated into a single SES Considerations section that includes the 3rd risk management property, Effectiveness, and is tied to meeting the requirements of specific standards.
```

Comment:
`SES Considerations` or `SES considerations` - should be consistent accross the text